### DISCUSSION TITLE
Bug pour enregistrer

### DISCUSSION THREAD

**Discussion by Alpi-Web** [PrestaSafe/prettyblocks#175] [Q&A]
Bonjour,
Je suis en train de tester votre Builder (super boulot !) mais j'ai un bug je n'arrive pas à enregistrer mes modifs de textes avec le block Custom Text alors que cela fonctionne quand j'utilise le bloc FAQ. Avez vous une idée d'où pourrai venir le problème ?
Pour info j'ai installé la dernière version Prettyblocks 3.0.4 (+ Classicblock) sur un PS 8.0.5 en localhost.
![Capture d'écran 2023-12-04 110007](https://github.com/PrestaSafe/prettyblocks/assets/129090053/e35d1561-9534-4838-a32f-3402e774dfac)

Merci

**Comment by PrestaSafe** [comment]
Bonjour, et merci du retour, pouvez vous voir si dans la console javascript vous n'avez pas une erreur 500, je vous invite également à vérifier les données dnas la table des blocks de prettyblocks si ce dernier n'est pas enregistrer. 
Cordialement

**Reply by Alpi-Web** [reply]
Merci pour ce retour rapide !
Alors voila le screenshot de ma console du coup pas d'erreur 500
![image](https://github.com/PrestaSafe/prettyblocks/assets/129090053/bc3b4e67-d020-421d-b229-36d9bb3f61f9)

sinon je vois bien le bloc dans la table mais je ne vois pas où est stocké son contenu
![image](https://github.com/PrestaSafe/prettyblocks/assets/129090053/57b49191-93e1-4530-bd97-619e3892e844)

**Reply by Alpi-Web** [reply]
Et voila la ligne en erreur
t.console.log("JQMIGRATE: Migrate is installed" + (e.migrateMute ? "" : " with logging active") + ", version " + e.migrateVersion));

**Reply by PrestaSafe** [reply]
Avez vous mis un `code`  à votre block ?

**Reply by Alpi-Web** [reply]
Non

**Reply by PrestaSafe** [reply]
Il le faut, c'est obligatoire :) 
Voici le détail dans la documentation: 
https://prettyblocks.io/docs/3/get-started/create-a-block

**Reply by Alpi-Web** [reply]
Ok mais je n'ai pas créé de blocs custom, j'ai simplement installé Classicblocs

**Reply by PrestaSafe** [reply]
Ah mince, autant pour moi, je vais regarder ça en détails. 
Cordialement

**Reply by Alpi-Web** [reply]
Aussi, quand je clique sur le bloc il y a furtivement une alerte qui s'affiche sur l'éditeur TINY "A valid API Key is required starting 2024..." mais bon vu que j'arrive à enregister du contenu sur le bloc FAQ cela me semblerait bizarre que ce soit ça

**Reply by PrestaSafe** [reply]
Non ce n'est pas lié à cela, vous avez 2 entrées en base de données sans code et sans zones, je vous invite à les supprimer dans un premier temps. Je vais tenter de creuser ça aujourdh'ui. 
Cordialement

**Reply by Alpi-Web** [reply]
Oui c'était des tests, je viens de supprimer mais toujours pas de changement pour enregistrer le contenu. Etant donné que cela fonctionne sur la FAQ, cela ne pourrai a pas être lié au module natifs de PS car cela bug aussi quand je veux ajouter une bannière

**Reply by Alpi-Web** [reply]
Pour info je viens de tester sur un PS 1.7.8.10 de test et j'ai toujours le même bug
https://wildwildwild.fr/

**Reply by Alpi-Web** [reply]
Bonjour, avez vous plus d'infos à propos de mon bug ? Merci ;)
